apps/wallet/src/ui/app/hooks/useObjectsOwnedByAddress.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useQuery } from '@tanstack/react-query';
+
+import { useRpc } from '_hooks';
+
+import type { SuiAddress } from '@mysten/sui.js';
+
+export function useObjectsOwnedByAddress(address?: SuiAddress | null) {
+    const rpc = useRpc();
+    return useQuery(
+        ['objects-owned', address],
+        () => rpc.getObjectsOwnedByAddress(address!),
```

Comment:
I thought we were trying to avoid fetching all owned objects. Is there a way to avoid this?

Reply:
Yes, except for this section.